### PR TITLE
tldr: edit examples

### PR DESCRIPTION
**Pull request by marchersimon**
I also wanted to add examples for `--theme` and `--search`, but I'm not sure if other clients also support that.

### CODE REVIEW

**Comment by bl-ue** [comment]
@marchersimon does this have anything to do with #5401?

**Comment by sbrl** [comment]
Hey there, @marchersimon! Thanks for the edit. Unfortunately, this is one of the very few pages that we can't use long-form arguments on, because they are optional in our [client spec](https://github.com/tldr-pages/tldr/blob/master/CLIENT-SPECIFICATION.md).

**Comment by marchersimon** [comment]
@bl-ue not on purpose, but it seems like this addresses that issue. However there already is a PR (#5443) for that. Should we adapt the changes of that PR here and close it or should we wait for it to be merged?

@sbrl oh, sure, I'll change it

**Comment by marchersimon** [comment]
Btw, why is there a `-p` option? There isn't such an option for the node client, instead the option for changing the platform is `-o`/`--os`.

**Comment by sbrl** [comment]
`-p` is what's specified in the [client spec](https://github.com/tldr-pages/tldr/blob/master/CLIENT-SPECIFICATION.md#arguments) - it's short for `--platform` @marchersimon

Sounds like that's a spec compliance issue with the Node.js client - great catch!

Would you like to open an issue over there about it?

**Comment by marchersimon** [comment]
Sure

File: pages/common/tldr.md [hunk 1]
```diff
@@ -3,15 +3,15 @@
...
-- Get typical usages of a command (hint: this is how you got here!):
+- Show the tldr page for a command (hint: this is how you got here!):
 
 `tldr {{command}}`
 
-- Show the tar tldr page for Linux:
+- Show the tldr page for `cd`, overriding the default platform:
 
-`tldr -p {{linux}} {{tar}}`
+`tldr -o {{windows}} {{cd}}`
```
Comment (bl-ue) [inline pages/common/tldr.md:12]:
```suggestion
`tldr -p {{windows}} {{cd}}`
```
https://github.com/tldr-pages/tldr-node-client/pull/337

Comment (marchersimon) [reply pages/common/tldr.md:12]: There was a reason, we don't use `--platform` and `--update` here, right?

Comment (bl-ue) [reply pages/common/tldr.md:12]: Yep: https://github.com/tldr-pages/tldr/pull/5508#issuecomment-808312905

Comment (marchersimon) [reply pages/common/tldr.md:12]: Ah, I knew there was something.

**Review by sbrl — APPROVED** [review]
Thanks, @marchersimon!

**Comment by sbrl** [comment]
Trivia: The `windows` platform was added much later than the other platforms. It's now the 2nd newest platform after the recent `android` addition.

**Review by bl-ue — APPROVED** [review]

**Pull request merged**